src/application-base/ApplicationBase.jsx

```diff
@@ -100,36 +99,40 @@ const ApplicationBase = ({
     };
   }, [unloadPromptIsDisabled, registeredPromptsRef]);
 
+  const theme = useMemo(() => ({ className: themeName }), [themeName]);
+
   return (
-    <ThemeProvider
-      className={cx('application-theme-provider', { fill: !fitToParentIsDisabled })}
-      themeName={themeName}
-      isGlobalTheme={themeIsGlobal}
-    >
-      <Base
-        customMessages={customTranslatedMessages}
-        translationsLoadingPlaceholder={translationsLoadingPlaceholder}
-        locale={locale || browserLocale}
+    <div className={cx('application-base', { fill: !fitToParentIsDisabled })}>
+      <ThemeProvider
```

Comment:
Its almost too bad we updated this to be a react fragment

Reply:
Almost, In the next MVB for terra-application I think we'll remove `fitToParentIsDisabled ` So we may still be able to kill this div.